meetings/minutes/2022-09-12-TSC-Minutes.md

```diff
@@ -0,0 +1,93 @@
+# Minutes from the 2022-09-12 Laminas Technical Steering Committee Meeting
+
+- When: 2022-00-12 at 19:00 UTC, until 20:36 UTC
+- Where: Laminas Slack #tsc-meeting channel
+- Attending:
+  - Abdul Malik Ikhsan
+  - Aleksei Khudiakov
+  - Andreas Heigl (left after second agenda item)
+  - Frank
+  - Gary Lockett
+  - George Steel
+  - Luís Cobucci
+  - Marco Pivetta (left after second agenda item)
+  - Matthew Weier O'Phinney
+
+Quorum WAS met (9 out of 15 members were present).
+
+## Agenda
+
+### Default git branching for new majors
+
+When creating a new major, the relevant branch is created; it is also set as the default branch for the repository. However, there can be a significant gap between branch creation and the first release of that major version.
+
+Contributors and renovate, will attempt to make bugfixes to this new major when they equally apply to the previous (current latest release) major.
+
+I (ed: George Steel) propose that the default branch should only be changed to the new major branch, once a release has been tagged for that new major.
+
+#### Discussion
+
+This had no debate; anybody who has had to maintain a repo that has an unreleased new major branch has run into the issue.
+Matthew noted that we need automatic-releases to create the new minor branch when a new major branch already exists, as that's the root cause of the problem.
+Aleksei noted that the logic should be "if the newly created branch version is higher than the current default branch, set the default to the new branch; otherwise, don't."
+
+Matthew volunteered to create an issue on the automatic-releases repo to request this.
+
+### Security only packages and dependency updates
+
+Is this intention for security only packages to eventually be abandoned? If no, then should we (I) look at ways to allow renovate to keep these up to date, if only to give us warning when the package may actually need maintenance. Currently any PR would be automatically closed, including the "Configure Renovate" PRs.
```

Comment:
forgot to add my name in the original PR here too, sorry

Reply:
Added attribution